src/css/main.less

```diff
@@ -61,6 +62,7 @@ a:focus {
 a,
 .btn {
   transition: all 0.2s ease-in-out;
+  padding: 7px 15px;
```

Comment:
good point about the `@font-size-h2` suggestion  – not too sure about `@container-fluid-padding` in this context (it's the `btn`'s own padding). I don't see value in creating a new variable for that, as well